src/cloffeine/common.clj

```diff
@@ -1,12 +1,16 @@
 (ns cloffeine.common
-  (:import [com.github.benmanes.caffeine.cache Caffeine CacheLoader AsyncCacheLoader Cache CacheWriter
-            Weigher]
+  (:import [com.github.benmanes.caffeine.cache AsyncCacheLoader
+                                               Cache
+                                               CacheLoader
+                                               CacheWriter
+                                               Caffeine
+                                               Weigher]
            [com.github.benmanes.caffeine.cache.stats CacheStats]
-           [java.util.function Function BiFunction]
+           [java.util.function BiFunction Function]
```

Comment:
those changes are making the review a lot harder... maybe you can configure your IDE to avoid them>

Reply:
And I like to sort Java imports as IntelliJ sorts Java imports in Java source code. :)